service/policy/objects.proto

```diff
@@ -367,18 +360,87 @@ message PublicKey {
 
   oneof public_key {
     // kas public key url - optional since can also be retrieved via public key
-    string remote = 1 [ (buf.validate.field).cel = {
-      id : "uri_format"
-      message : "URI must be a valid URL (e.g., 'https://demo.com/') followed "
-                "by additional segments. Each segment must start and end with "
-                "an alphanumeric character, can contain hyphens, alphanumeric "
-                "characters, and slashes."
-      expression : "this.matches('^https://"
-                   "[a-zA-Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?(\\\\.[a-zA-"
-                   "Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?)*(/.*)?$')"
-    } ];
+    string remote = 1 [(buf.validate.field).cel = {
+      id: "uri_format"
+      message:
+        "URI must be a valid URL (e.g., 'https://demo.com/') followed "
+        "by additional segments. Each segment must start and end with "
+        "an alphanumeric character, can contain hyphens, alphanumeric "
+        "characters, and slashes."
+      expression:
+        "this.matches('^https://"
+        "[a-zA-Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?(\\\\.[a-zA-"
+        "Z0-9]([a-zA-Z0-9\\\\-]{0,61}[a-zA-Z0-9])?)*(/.*)?$')"
+    }];
 
     // public key with additional information. Current preferred version
     KasPublicKeySet cached = 3;
   }
 }
+
+/*
+   Key Management
+*/
+
+message ProviderConfig {
```

Comment:
KeyProviderConfig?

Reply:
I will update this. I think this makes sense now that I moved it to objects.proto to remove the circular dependencies.